### PR TITLE
PLAT-126939: Separate labels for navigation buttons in WizardPanels

### PR DESCRIPTION
**Pull request by 0x64**
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
GUI requires to show button labels separated from icon buttons.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Resize focus effects' size and update labels' text color on press to match latest designs.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-126939

### CODE REVIEW

File: WizardPanels/WizardPanels.js [hunk 1]
```diff
@@ -389,6 +389,7 @@ const WizardPanelsBase = kind({
 							aria-label={$L('Previous')}
 							backgroundOpacity="transparent"
 							component={prevButton}
+							focusIconOnly
```
Comment (jeonghee27) [inline WizardPanels/WizardPanels.js:392]: `separateLabel` ? I think @MikyungKim  will decide the prop name.

**Comment by jeonghee27** [comment]
There is no screenshot tc for buttons with label

**Review by jeonghee27 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/PopupTabLayout` to collapse its tab only when a user enters a menu
 - `sandstone/Scroller` focus rule to match latest UX when `focusableScrollbar` prop is `byEnter`
 - `sandstone/Scroller` and `sandstone/VirtualList` to hide the scrollbar after N seconds
+- `sandstone/WizardPanels.Panel` buttons to show labels separately to match latest designs
```
Comment (MikyungKim) [inline CHANGELOG.md:22]:
```suggestion
- `sandstone/WizardPanels.Panel` `nextButton` and `prevButton` to show labels separately to match latest designs
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**